src/instruments/src/EFB/Dispatch/Dispatch.tsx

```diff
@@ -47,32 +47,24 @@ type DispatchProps = {
     taxiOutTime: number,
 };
 
-type DispatchState = {
-    activeIndex: number,
-};
+const Dispatch = (props: DispatchProps) => {
+    const [activeIndex, setActiveIndex] = useState(0);
 
-class Dispatch extends React.Component<DispatchProps, DispatchState> {
-    tabs = [
+    const tabs = [
         'Overview',
         'OFP',
         'Fuel',
     ];
 
-    constructor(props: DispatchProps) {
-        super(props);
-        this.handleClick = this.handleClick.bind(this);
-        this.state = { activeIndex: 0 };
-    }
-
-    handleClick(index: number) {
-        this.setState({ activeIndex: index });
+    function handleClick(index: number) {
```

Comment:
since we're not using `this` anymore maybe switch these to arrow functions? Functionality remains the same though so it's not necessary.

Reply:
these can remain as-is